frontend/app/routes/$lang/_protected/applications/$id.tsx

```diff
@@ -12,23 +12,28 @@ import { getAuditService } from '~/services/audit-service.server';
 import { getInstrumentationService } from '~/services/instrumentation-service.server';
 import { getLookupService } from '~/services/lookup-service.server';
 import { getRaoidcService } from '~/services/raoidc-service.server';
-import { parseDateTimeString, toLocaleDateString } from '~/utils/date-utils';
+import { extractDateParts, parseDateTimeString, toLocaleDateString } from '~/utils/date-utils';
 import { featureEnabled } from '~/utils/env.server';
 import { getNameByLanguage, getTypedI18nNamespaces } from '~/utils/locale-utils';
 import { getFixedT, getLocale } from '~/utils/locale-utils.server';
 import { localizeMaritalStatuses } from '~/utils/lookup-utils.server';
+import { mergeMeta } from '~/utils/meta-utils';
 import { IdToken } from '~/utils/raoidc-utils.server';
 import { RouteHandleData } from '~/utils/route-utils';
+import { getTitleMetaTags } from '~/utils/seo-utils';
 import { formatSin } from '~/utils/sin-utils';
 import { useUserOrigin } from '~/utils/user-origin-utils';
 
 export const handle = {
-  breadcrumbs: [{ labelI18nKey: 'applications:view-application.page-title' }],
```

Comment:
Shouldn't this page have breadcrumbs to the main applications page?

Reply:
Wasn't sure because of the Figmas. Updated: 9323cab4f704052c540a8799beb30a2f0a651956